### DISCUSSION TITLE
Is there an online Whisper instance I could use programmatically / with an API?

### DISCUSSION THREAD

**Discussion by MoltenMonster** [openai/whisper#1622] [Q&A]
I know that Whisper needs lots of computing power to run and to run it online would be to use someone else's computer(s) but I thought I might as well ask. It takes a very long time on my M1 Mac, one I found is the [Whisper JAX Huggingface Space](https://huggingface.co/spaces/sanchit-gandhi/whisper-jax) which is very fast but there's no API to use it, I can only use the Gradio UI. I want to be able to upload recordings programmatically and get back text files with timestamps.

**Comment by phineas-pta** [comment]
online service: https://platform.openai.com/docs/guides/speech-to-text

for apple silicon try https://github.com/ggerganov/whisper.cpp
